source/règles/base.yaml

```
@@ -780,40 +777,72 @@
   références:
     Le salaire. Fixation et paiement: http://travail-emploi.gouv.fr/droit-du-travail/remuneration-et-participation-financiere/remuneration/article/le-salaire-fixation-et-paiement
 
-- espace: contrat salarié . salaire . brut de base
+- espace: contrat salarié . rémunération . brut de base
   nom: équivalent temps plein
-  applicable si: temps partiel
+  applicable si: temps de travail . temps partiel
   titre: Salaire brut équivalent temps plein
   résumé: Le salaire si l'embauche se faisait à temps plein
   question: Quel est le salaire en équivalent temps plein ?
   unité: €
-  formule: brut de base / quotité de travail
+  formule: brut de base / temps de travail . quotité de travail
   période: flexible
   suggestions:
     salaire médian: 2300
     SMIC: 1522
 
-- espace: contrat salarié . salaire
-  nom: brut
-  titre: Salaire brut
+- espace: contrat salarié . rémunération . heures supplémentaires
+  nom: taux horaire
+  description: Le taux horaire utilisé pour calculer la rémunération liée au heures supplémentaires. Il intègre le
+  période: aucune
```

```suggestion
  période: flexible
```

corrige le problème (mais c'est vrai que le taux horaire ne dépend pas de la période...)

---

J'ai utilisé le modificateur temporel [mensuel] sur les valeurs du numérateur.